Unit testing edits (#36339)
dotnet/docs: docs/azure/sdk/unit-testing-mocking.md

};
```

> To create instances of output models, a model factory is used. Azure SDK client libraries provide a static model factory class that ends in `ModelFactory` and contains a set of static methods to initialize the library's output model types.

Reviewer: Not sure if this helps frame things?

```suggestion
To create instances of output models, a model factory is used. Azure SDK client libraries provide a static model factory class that ends in `ModelFactory` and contains a set of static methods to initialize the library's output model types. For example, the model factory for `SecretClient` is `SecretModelFactory`:
```

> To create instances of output models, a model factory is used. Azure SDK client libraries provide a static model factory class that ends in `ModelFactory` and contains a set of static methods to initialize the library's output model types. For example, the model factory for `SecretClient` is `SecretModelFactory`:

Reviewer 2: ```suggestion
To create instances of output models, a model factory is used. Azure SDK client libraries provide a static model factory class with a `ModelFactory` suffix in its name. The class contains a set of static methods to initialize the library's output model types. For example, the model factory for `SecretClient` is `SecretModelFactory`:
```
